.github/workflows/publish-on-tag.yml

```
@@ -37,3 +34,19 @@ jobs:
           npm config set registry 'https://wombat-dressing-room.appspot.com/'
           npm config set '//wombat-dressing-room.appspot.com/:_authToken' '${NPM_TOKEN}'
           npm publish
+
+  post-publish:
+    needs: publish
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Bump to Post Version
+        run: |
+          VERSION=$(node utils/apply_post_version.js)
+          npm run build
+          npm run doc
+          git config user.name 'Git Actions'
+          git config user.email '<>'
+          git commit -m 'chore: bump version to v$VERSION'"
```

In Bash, single-quoted strings don’t interpolate. I think this is what you want:

```suggestion
          git commit -m "chore: bump version to v${VERSION}"
```

---

Done!